### PR TITLE
[HOTFIX] Dynamic form in python interpreter

### PR DESCRIPTION
**Pull request by Leemoonsoo**
### What is this PR for?
https://github.com/apache/zeppelin/pull/2106 rewrote python interpreter. But dynamic form feature is not rewritten correctly.

### What type of PR is it?
Hot Fix

### Todos
* [x] - Bring dynamic form back

### What is the Jira issue?
https://github.com/apache/zeppelin/pull/2106 

### How should this be tested?
run

```
%python
print("Hello "+z.input("name", "sun"))
```

```
%python
print("Hello "+z.select("day", [("1","mon"),
                                ("2","tue"),
                                ("3","wed"),
                                ("4","thurs"),
                                ("5","fri"),
                                ("6","sat"),
                                ("7","sun")]))
```

```
%python
options = [("apple","Apple"), ("banana","Banana"), ("orange","Orange")]
print("Hello "+ " and ".join(z.checkbox("fruit", options, ["apple"])))
```

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: interpreter/lib/python/backend_zinline.py [hunk 1]
```diff
@@ -94,7 +95,10 @@ def get_bytes(self, **kwargs):
         buf = BytesIO()
         self.print_figure(buf, **kwargs)
         fmt = fmt.encode()
-        byte_str = b"data:image/%s;base64," %fmt
+        if sys.version_info >= (3, 4) and sys.version_info < (3, 5):
+            byte_str = bytes("data:image/%s;base64," %fmt, "utf-8")
+        else:
+            byte_str = b"data:image/%s;base64," %fmt
```
Comment (felixcheung) [inline interpreter/lib/python/backend_zinline.py]: do we need to be concerned with Python 2.x here?

Comment (Leemoonsoo) [inline interpreter/lib/python/backend_zinline.py]: python 2.x works fine. 3.4 is only exception as far as i know.

**Comment by AhyoungRyu** [comment]
Tested both with & without this patch. It works well now.

**Comment by Leemoonsoo** [comment]
Merge to master and branch-0.7 as a hotfix.

**Pull request closed without merging**